src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
I ended up adding the following to src\Common\src\Interop\Unix\System.Native\Interop.Socket.cs and src\Common\src\System\Net\SocketProtocolSupportPal.Unix.cs 

#pragma warning disable 0436
using ProtocolType = System.Net.Internals.ProtocolType;
using SocketType = System.Net.Internals.SocketType;